python/Upload_CV_List.py

```diff
@@ -0,0 +1,31 @@
+# Preliminaries
+import pandas as pd
+import argparse
+import synapseclient
+from synapseclient import Project, File, Folder
+from synapseclient import Schema, Column, Table, build_table, Row, RowSet, as_table_columns
+
+syn = synapseclient.Synapse()
+syn.login()
+
+# Get arguments
+parser = argparse.ArgumentParser(description='Get synapse project id and filepath')
+parser.add_argument("--p", "--project_id", help="Synapse Project ID")
+# parser.add_argument("--f", "--filepath", help="path to access file")
```

Comment:
This command-line argument is currently not being used. Utilize this argument instead of the hard-coded path on L25.